src/app/signin/page.tsx

```
@@ -1,29 +1,106 @@
+// =============================================================================
+interface SigninFormData {
   email: string;
   password: string;
   rememberMe: boolean;
+}
+
+// =============================================================================
+// GOOGLE ICON COMPONENT
+// =============================================================================
+const GoogleIcon = () => (
+  <svg className="h-5 w-5" viewBox="0 0 48 48">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)